app/src/main/res/layout/fragment_map_page.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This linear layout is used to show elements
+     in vertical or in horizontal linear manner -->
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:gravity="center">
+
+    <!-- This TextView indicates new fragment is open -->
```

Comment:
Was there a TextView before? Maybe remove this comment since it doesn't seem to exist anymore